userguide/platform/userguide_configuration.adoc

```diff
@@ -388,3 +248,39 @@ In case of a https://docs.killbill.io/latest/development.html#_running_the_appli
+
+The following table lists the Kaui Configuration properties. 
+
+[options="header",cols="1,1,1,1,1"]
+|===
+|Category   |Property Name   |Environment Variable   |Description   |Default Value   
+//----------------------------------------
+|Database   |kaui.db.adapter   |KAUI_DB_ADAPTER   |Specifies the Kaui Database Adapter   |mysql2   
+|Database   |kaui.db.encoding   |KAUI_DB_ENCODING   |Specifies the Kaui Database Encoding   |utf8   
+|Database   |kaui.db.database   |KAUI_DB_DATABASE   |Specifies the Kaui Database Encoding (only applicable if `kaui.db.adapter=sqlite3`)   |/var/tmp/kaui.sqlite3   
+|Database   |kaui.db.url   |KAUI_DB_URL   |Specifies the Kaui Database URL   |jdbc:mysql://localhost:3306/kaui?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC   
+|Database   |kaui.db.username   |KAUI_DB_USERNAME   |Specifies the Kaui Database Username   |root   
+|Database   | kaui.db.password  |KAUI_DB_PASSWORD   |Specifies the Kaui Database Password   |root   
+|Database   |kaui.db.host   |KAUI_DB_HOST   |Specifies the Kaui Database Host   |None   
+|Database   |kaui.db.port   |KAUI_DB_PORT   |Specifies the Kaui Database Port   |None   
+|Database   |kaui.db.pool   |KAUI_DB_POOL   |Coming Soon   |50   
+|Database   |kaui.db.timeout   |KAUI_DB_TIMEOUT   |Coming Soon   |5000   
+|Kill Bill Client   |kaui.url   |KILLBILL_URL/KAUI_KILLBILL_URL   |Specifies the Kill Bill URL   |http://127.0.0.1:8080   
+|Kill Bill Client|kaui.disable_ssl_verification   |KILLBILL_DISABLE_SSL_VERIFICATION/KAUI_KILLBILL_DISABLE_SSL_VERIFICATION   |Coming Soon   |Coming Soon   
+|Kill Bill Client   |kaui.read_timeout   |KILLBILL_READ_TIMEOUT/KAUI_KILLBILL_READ_TIMEOUT   |Coming Soon   |60000   
+|Kill Bill Client   |kaui.connection_timeout   |KILLBILL_CONNECTION_TIMEOUT/KAUI_KILLBILL_CONNECTION_TIMEOUT   |Coming Soon   |60000   
+|Kaui   |kaui.demo   |KAUI_DEMO_MODE   |Coming Soon   |Coming Soon   
+|Kaui   |kaui.plugins_whitelist   |KAUI_PLUGINS_WHITELIST   |Coming Soon   |None   
+|Kaui   |kaui.root_username   |KAUI_ROOT_USERNAME   |Coming Soon   |admin   
+|Kaui   |kaui.disable_sign_up_link   |KAUI_DISABLE_SIGN_UP_LINK   |Coming Soon   |Coming Soon   
+|Kaui   |kaui.chargeback_reason_codes   |KAUI_CHARGEBACK_REASON_CODES   |Coming Soon   |None   
+|Kaui   |kaui.credit_reason_codes   |KAUI_CREDIT_REASON_CODES   |Coming Soon   |None   
+|Kaui   |kaui.invoice_item_reason_codes   |KAUI_INVOICE_ITEM_REASON_CODES   |Coming Soon   |None 
+|Kaui   |kaui.invoice_payment_reason_codes   |KAUI_INVOICE_PAYMENT_REASON_CODES   |Coming Soon   |None 
+|Kaui   |kaui.payment_reason_codes | KAUI_PAYMENT_REASON_CODES  |Coming Soon   |None   
+|Kaui   |kaui.refund_reason_codes   |KAUI_REFUND_REASON_CODES   |Coming Soon   |None 
+|Encryption   |kaui.encryption.filename   |KAUI_ENCRYPTION_FILENAME   |Coming Soon   |config/symmetric-encryption.yml 
+|===
```

Comment:
I will need some help with some of the property descriptions. I have added the text "Coming Soon" in the meantime :)

Reply:
Sounds good, we can chat about this on Slack :-) 

In case it helps, check https://github.com/killbill/killbill-admin-ui-standalone/blob/master/CUSTOMIZATION.md which explains how to fully customize Kaui via code. The code snippets have some lightweight descriptions.

FWIW This is where some of these are used in the code, to configure the dropdowns: https://github.com/killbill/killbill-admin-ui/blob/08f6c9abc7e68dc41e68f90b9500eeac1cc44f84/app/views/kaui/chargebacks/_form.html.erb#L15